packages/block-library/src/navigation/use-navigation-menu.js

```diff
@@ -71,8 +71,9 @@ function selectNavigationMenus( select ) {
 	const args = [
 		'postType',
 		'wp_navigation',
-		{ per_page: -1, status: [ 'publish', 'draft' ] },
+		{ per_page: -1, status: 'publish,draft' },
```

Comment:
@scruffian If we do this we will need a comment to explain why it _has_ to be a string for now.